connectivity/check/deployment.go

```diff
@@ -86,6 +86,11 @@ func newDeployment(p deploymentParameters) *appsv1.Deployment {
 							ImagePullPolicy: corev1.PullIfNotPresent,
 							Command:         p.Command,
 							ReadinessProbe:  p.ReadinessProbe,
+                            SecurityContext: &corev1.SecurityContext{
+                                Capabilities: &corev1.Capabilities{
+                                    Add: []corev1.Capability{"NET_RAW"},
+                                },
+                            },
```

Comment:
The indentations and/or closing brackets are somehow messed up.

Reply:
Fixed now. Sorry about that. My editor and me usually live in a spaces world ... ;-)